mix.exs

```diff
@@ -31,7 +31,7 @@ defmodule Nerves.Runtime.MixProject do
     [
       {:system_registry, "~> 0.5"},
       {:elixir_make, "~> 0.4", runtime: false},
-      {:ex_doc, "~> 0.18.0", only: [:dev, :test], runtime: false},
+      {:ex_doc, "~> 0.19", only: [:dev, :test], runtime: false},
```

Comment:
Doesn't this break Elixir 1.6 users who have projects that depend on `nerves_runtime` and also use `ex_doc`?

Reply:
`~> 0.18` would be all we really need, I think.